_config.yml

```diff
@@ -34,6 +40,19 @@ defaults:
     values:
       layout: "tour"
       permalink: "/tour/:basename/"
+  -
+    scope:
+      path: ""
+      type: "fluo-docs-1-2"
+    values:
+      layout: "fluo-docs-1.2"
+      title_prefix: "Fluo Documentation - "
+      version: "1.2.0"
+      minor_release: "1.2"
+      docs_base: "/docs/fluo/1.2"
+      javadoc_base: "https://static.javadoc.io/org.apache.fluo/fluo-api/1.1.0-incubating"
+      github_base: "https://github.com/apache/fluo/blob/master"
```

Comment:
Should this be a tag?  The 1.2 tag eventually?  If so, maybe we can open an issue or set it now?

Reply:
Created #88